app.js

```diff
@@ -49,12 +51,13 @@ if ( app.get( 'env' ) === 'development' ) {
 
 // production error handler
 // no stacktraces leaked to user
-app.use( function( err, req, res, next ) {
-  res.status( err.status || 500 )
-  res.send({
-    message: err.message,
-    error: {}
+  app.use( (err, req, res, next) => {
```

Comment:
Why are these lines changed? (let's revert this change so we're not adding diffs that are not related to our current feature implementation)